src/main/scala/no/ndla/searchapi/service/search/SearchConverterService.scala

```diff
@@ -141,6 +141,7 @@ trait SearchConverterService {
         "data-videoid",
         "data-url",
         "data-resource_id",
+        "data-content-id",
```

Comment:
Egentlig rart vi har brukt anna navn på parameteren for concepts. Kunne vurdert å endre det på sikt.